bigbluebutton-html5/imports/startup/client/intl.jsx

```diff
@@ -140,11 +151,6 @@ class IntlStartup extends Component {
               this.setState({ messages: mergedMessages, fetching: false, normalizedLocale: dasherizedLocale }, () => {
                 IntlStartup.saveLocale(dasherizedLocale);
               });
-            })
-            .catch(() => {
-              this.setState({ fetching: false, normalizedLocale: null }, () => {
-                IntlStartup.saveLocale(DEFAULT_LANGUAGE);
```

Comment:
Do we need to .saveLocale somewhere else?

Reply:
not anymore... after removing the one in .catch, there's only one .saveLocale call left